### PR TITLE
Update stones spec

### PR DESCRIPTION
**Pull request by matiasfmartino**
Now the output is finally structured :D

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
 require 'codeclimate-test-reporter'
 
+require_relative '../lib/gobstones_server'
```
Comment (faloi) [inline spec/spec_helper.rb]: Why is this necessary?

Comment (matiasfmartino) [inline spec/spec_helper.rb]: because a conflict between ActiveSupport::JSON and the JSON gem. For more info ask @flbulgarelli :p

Comment (flbulgarelli) [inline spec/spec_helper.rb]: And because it is Ruby :stuck_out_tongue: You always need to require you files!
